cypress/integration/upload.spec.ts

```diff
@@ -39,7 +39,7 @@ describe("Upload from the the landing page", function () {
 
     cy.get("@download-kmp")
       .should("have.attr", "data-download-state", "ready")
-      .click();
+      .click({ force: true });
```

Comment:
The cypress test failed and the error message said cy.click() failed because this element is not visible. I add force:true to ignore it... I don't know why this test failed, it always passed before.